src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/help/Get-AzureRmDataLakeAnalyticsJobPipeline.md

```diff
@@ -110,7 +115,6 @@ System.Guid
 
 ## OUTPUTS
 
-### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.DataLakeAnalytics.Models.PSJobPipelineInformation, Microsoft.Azure.Commands.DataLakeAnalytics, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
@TaraMeyer minor comment, but instead of removing this header, could we just change it to `Microsoft.Azure.Commands.DataLakeAnalytics.Models.PSJobPipelineInformation` and remove the line below?